### DISCUSSION TITLE
Timed out waiting for all PodVolumeRestores to complete

### DISCUSSION THREAD

**Discussion by Jedi-5** [vmware-tanzu/velero#7704] [Community support Q&A]
While restore of Cloudbees cjoc following error occurs, but backup is successful.
Velero: v1.13.0

### **Backup**

Phase:  Completed

Resources:
  Included:        *
  Excluded:        <none>
  Cluster-scoped:  auto

Storage Location:  default

Velero-Native Snapshot PVs:  auto
Snapshot Move Data:          false
Data Mover:                  velero

CSISnapshotTimeout:    10m0s
ItemOperationTimeout:  30m0s

Backup Format Version:  1.1.0

Started:    2024-04-18 19:19:19 +0000 UTC
Completed:  2024-04-18 19:26:19 +0000 UTC

Expiration:  2024-05-18 19:19:19 +0000 UTC

**Total items to be backed up:  215
Items backed up:              215**

Backup Volumes:
  Velero-Native Snapshots: <none included>

  CSI Snapshots: <none included>

  Pod Volume Backups - kopia:
    Completed:
      cloudbees-sda/cb-flow-bound-agent-flow-agent-0: flow-bound-agent-workspace
      cloudbees-sda/cjoc-0: jenkins-home, tmp
      cloudbees-sda/flow-devopsinsight-0: elasticsearch-data
      cloudbees-sda/flow-server-777f77b6b-hsjrk: plugins-volume
      cloudbees-sda/mariadb-0: data
      cloudbees-sda/nfs-release-cloudbees-nfs-subdir-external-provisioner-6cdbsbwmp: nfs-subdir-external-provisioner-root
      lob-caas/caas-0: jenkins-home, tmp
      lob-test2/test2-0: jenkins-home, tmp
      lob-test3/test3-0: jenkins-home, tmp
      lob-test4/test4-0: jenkins-home, tmp
      lob-test5/test5-0: jenkins-home, tmp

### **Restore**

Phase:                       PartiallyFailed

**Total items to be restored:  215
Items restored:              215**

Started:    2024-04-18 19:33:34 +0000 UTC
Completed:  2024-04-18 20:03:34 +0000 UTC

Errors:
  Velero:   **timed out waiting for all PodVolumeRestores to complete**
  Cluster:    <none>
  Namespaces: <none>

Namespaces:
  Included:  all namespaces found in the backup
  Excluded:  <none>

Resources:
  Included:        *
  Excluded:        nodes, events, events.events.k8s.io, backups.velero.io, restores.velero.io, resticrepositories.velero.io, csinodes.storage.k8s.io, volumeattachments.storage.k8s.io, backuprepositories.velero.io
  Cluster-scoped:  auto

Restore PVs:  auto

kopia Restores:
  Completed:
    cloudbees-sda/cjoc-0: jenkins-home, tmp
    cloudbees-sda/flow-server-777f77b6b-hsjrk: plugins-volume
    cloudbees-sda/mariadb-0: data
    cloudbees-sda/nfs-release-cloudbees-nfs-subdir-external-provisioner-6cdbsbwmp: nfs-subdir-external-provisioner-root
    lob-caas/caas-0: jenkins-home, tmp
    lob-test2/test2-0: jenkins-home, tmp
    lob-test3/test3-0: jenkins-home, tmp
    lob-test4/test4-0: jenkins-home, tmp
    lob-test5/test5-0: jenkins-home, tmp
  New:
    **cloudbees-sda/flow-devopsinsight-0: elasticsearch-data**

Existing Resource Policy:   <none>
ItemOperationTimeout:       30m0s

Preserve Service NodePorts:  auto

Please guide on resolving the above issue.

**Comment by kaovilai** [comment]
Related issues:

https://github.com/vmware-tanzu/velero/issues/4917

Can you [generate debug bundle](https://velero.io/docs/v1.13/troubleshooting/#:~:text=can%20use%20the-,velero%20debug,-command%20to%20generate) for us?

**Reply by Jedi-5** [reply]
Thanks @kaovilai for quick reply!
Sorry to say, not possible to share the detailed debug bundle due to confidentiality of sharing corporate data.

Error "timed out waiting for all PodVolumeRestores to complete" is specific to "cloudbees-sda/flow-devopsinsight-0: elasticsearch-data".
